contracts/libs/Tx.sol

```diff
@@ -173,7 +173,7 @@ library Tx {
                 break;
             }
         }
-        require(x < 0x0fff, "Bad input");
+        require(x <= 0x0fff, "Bad input");
```

Comment:
So the equality case 0x0fff is 4095, it's still a good mantissa.

Reply:
Great catch